rules/whole-team-quality/rule.md

```diff
@@ -16,7 +16,7 @@ If you have a dedicated tester embedded in the Scrum team, they are not solely r
 
 <!--endintro-->
 
-![Figure: The whole team needs to take on responsibility to achieve great quality (source: 2017 SAP Quality Awards finalists © Carina Kircher Fotografie)](quality-award-winners.jpg)
```

Comment:
You need to delete that quality-award-winners.jpg now that it is not used anywhere

Reply:
Done - see https://github.com/SSWConsulting/SSW.Rules.Content/pull/5242

Reply:
Thanks. In future combine it into 1 PR (add to this)